einstein-ai/1.0/modules/ROOT/pages/einstein-ai-connector-troubleshooting.adoc

```diff
@@ -0,0 +1,5 @@
+= Troubleshooting Einstein Connector 1.0 - Mule 4
+
+== Empty API Instance URL Error
+
```

Comment:
as per our guidelines, we need to include a link to the Known Issues site in the See Alsos of troubleshooting pages

```
== See Also

* https://help.mulesoft.com[MuleSoft Help Center]
* https://issues.salesforce.com/[Known Issues]
```